contrib/buildrefactor/tests/python/pants_test/contrib/buildrefactor/buildozer_util.py

```diff
@@ -0,0 +1,28 @@
+# coding=utf-8
+# Copyright 2017 Pants project contributors (see CONTRIBUTORS.md).
+# Licensed under the Apache License, Version 2.0 (see LICENSE).
+
+from __future__ import (absolute_import, division, generators, nested_scopes, print_function,
+                        unicode_literals, with_statement)
+
+
+def assertInFile(self, string, file):
```

Comment:
`self` isn't used here eh? :P

and I think it would be good to push this up to some test utilities if you can.